src/Npgsql/TypeHandlers/NumericHandlers/DecimalRaw.cs

```diff
@@ -31,6 +31,8 @@ struct DecimalRaw
         // Do not change the order in which these fields are declared. It
         // should be same as in the System.Decimal struct.
         [FieldOffset(0)]
+        public decimal Value;
+        [FieldOffset(0)]
```

Comment:
Would it better to add a `public` property like for other fields or/and make it `readonly`?

Reply:
But we need to set it? Do you want to add an additional ctor as well?

Reply:
It depends on the second question. By the way, why do you ask this now if you t for `GuidRaw`? (:

Reply:
Because `GuidRaw` didn't already have a ctor taking a number, hence "additional ctor" 😉

Reply:
Personally I would prefer an additional `ctor` if it won't decrease perf.